### PR TITLE
slack and twitter embeds experiment [WIP]

### PR DESCRIPTION
**Pull request by juozasg**
Creating a preview to test metadata markup in slack and twitter.

### CODE REVIEW

File: layouts/partials/head.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+{{ partialCached "favicons.html" . }}
+<title>{{ if .IsHome }}{{ .Site.Title }}{{ else }}{{ with .Title }}{{ . }} | {{ end }}{{ .Site.Title }}{{ end }}</title>
+<meta name="description" content="{{ .Site.Params.Description }}">
+
+
+<meta property="og:title" content="Flux - the GitOps family of projects" />
+<meta property="og:description" content="<h1>Flux</h1> is a <strong>set</set> of <pre>continuous</pre> and progressive delivery solutions for Kubernetes, <h3>and</h3> they are open and extensible. 
+The APIs of Flux are stable now. Now is the best time to start using Flux, or start your migration if you are a legacy user." />
+<meta property="og:type" content="website" />
+<meta property="og:url" content="//localhost:1313/" /><meta property="og:image" content="//localhost:1313/img/flux-social.png"/>
```
Comment (dholbach) [inline layouts/partials/head.html:25]: Maybe link to `/` as opposed to localhost?

Comment (juozasg) [reply layouts/partials/head.html]: I pushed a change just to try it out, but I feel that slack and twitter probably will not render any markup in the op:description meta tag. I did not find references describing that. 

https://ogp.me/
https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup

Comment (juozasg) [reply layouts/partials/head.html]: With url fixes:
Screenshot from 2021-08-13 13-14-10
image

**Comment by juozasg** [comment]
I will delete this in a bit. See for details: https://github.com/fluxcd/website/issues/165#issuecomment-897957232

**Pull request closed without merging**